bramkouwen/Neatly-In-Motion
# Assets Folder Creation

## What does this PR do?
* It creates the assets folder
* It places content within the assets folder

## How do you check this PR?
* Pull down the branch
* Check the added assets folder 
* Check the added content

## Screenshots 
None.
